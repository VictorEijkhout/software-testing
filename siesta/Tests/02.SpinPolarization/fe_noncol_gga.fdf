%include ../base_fe3.fdf
SystemLabel         fe_noncol_gga

XC.Functional GGA
XC.Authors    PBE
Spin non-collinear

