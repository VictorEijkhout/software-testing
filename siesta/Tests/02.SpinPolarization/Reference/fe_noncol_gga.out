Executable      : /home/fnpedron/Siesta/siesta-dev/_build_serial/Src/siesta
Version         : 5.2.0-15-g8c00451d2
Architecture    : x86_64
Compiler version: GNU-11.4.0
Compiler flags  : -fallow-argument-mismatch -O3 -march=native
Environment variables: 
  SIESTA_PS_PATH: /home/fnpedron/Siesta/siesta-dev/Tests/Pseudos
Parallelisations: none
NetCDF support
NetCDF-4 support
Lua support
DFT-D3 support

Runtime information:
* Directory : /home/fnpedron/Siesta/siesta-dev/_build_serial/Tests/02.SpinPolarization/fe_noncol_gga/work
* Running in serial mode.
>> Start of run:  17-JAN-2025  12:35:01

                           *********************** 
                           *  WELCOME TO SIESTA  * 
                           *********************** 

 NOTE: Siesta V5 introduces some new defaults for basis-set generation
 NOTE: which might lead to slightly different numerical results.
 NOTE: Please see the manual for full details and compatibility measures.


reinit: Reading from file ../../fe_noncol_gga.fdf

reinit: -----------------------------------------------------------------------
reinit: System Name: Linear Fe3 cluster
reinit: -----------------------------------------------------------------------
reinit: System Label: fe_noncol_gga
reinit: -----------------------------------------------------------------------

NOTE on units for spin output: 
{S} is the spin magnetic moment, in units of the Bohr magneton,
assuming a g-factor for the electron spin of exactly 2.
The spin magnetic moment is then numerically equal to the charge inbalance,
in units of electrons, between the spin-up and -down channels.


initatom: Reading input for the pseudopotentials and atomic orbitals ----------
Species number:   1 Atomic number:   26 Label: Fe_nc

Ground state valence configuration (from tables):   4s02  4p00  3d06  4f00

---- Processing specs for species: Fe_nc
Reading pseudopotential information in formatted form from:
  /home/fnpedron/Siesta/siesta-dev/Tests/Pseudos/Fe_nc.psf

---- Pseudopotential check for Fe

Pseudized shells:
4s( 1.00) rc: 2.00
4p( 0.00) rc: 2.00
3d( 7.00) rc: 2.00
4f( 0.00) rc: 2.00
Valence configuration for ps generation: (assumed as above)
For Fe_nc, standard SIESTA heuristics set lmxkb to 3
 (one more than the basis l, including polarization orbitals).
Use PS.lmax or PS.KBprojectors blocks to override.

<basis_specs>
===============================================================================
Fe_nc                Z=  26    Mass=  55.845        Charge= 0.17977+309
Lmxo=2 Lmxkb= 3    BasisType=split      Semic=F
L=0  Nsemic=0  Cnfigmx=4
          i=1  nzeta=1  polorb=1  (4s)  (to be polarized perturbatively)
            splnorm:   0.15000    
               vcte:    40.000    
               rinn:  -0.90000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000    
            lambdas:    1.0000    
L=1  Nsemic=0  Cnfigmx=4
          i=1  nzeta=0  polorb=0  (4p)  (perturbative polarization orbital) (from 4s)
L=2  Nsemic=0  Cnfigmx=3
          i=1  nzeta=1  polorb=0  (3d)
            splnorm:   0.15000    
               vcte:    40.000    
               rinn:  -0.90000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000    
            lambdas:    1.0000    
-------------------------------------------------------------------------------
L=0  Nkbl=1  erefs: 0.17977+309
L=1  Nkbl=1  erefs: 0.17977+309
L=2  Nkbl=1  erefs: 0.17977+309
L=3  Nkbl=1  erefs: 0.17977+309
===============================================================================
</basis_specs>

Using PAO.SplitTailNorm
atom: Called for Fe_nc                 (Z =  26)
Maximum radius (at nrval) set to   50.00000 8518

read_vps: Pseudopotential generation method:
read_vps: ATM3      Troullier-Martins                       
Valence charge in pseudo generation:    8.00000

read_vps: Pseudopotential includes a core correction:
read_vps: Pseudo-core for xc-correction

xc_check: Exchange-correlation functional:
xc_check: GGA Perdew, Burke & Ernzerhof 1996
xc_check: WARNING: Pseudopotential generated with LDA CA functional
V l=0 = -2*Zval/r beyond r=  2.8472
V l=1 = -2*Zval/r beyond r=  2.8472
V l=2 = -2*Zval/r beyond r=  2.8472
V l=3 = -2*Zval/r beyond r=  2.8472
All V_l potentials equal beyond r=  1.9894
This should be close to max(r_c) in ps generation
All pots = -2*Zval/r beyond r=  2.8472
Using large-core scheme (fit) for Vlocal

atom: Estimated core radius    2.84716
Fit of Vlocal with continuous 2nd derivative
Fitting vlocal at       1.9954
Choosing vlocal chloc cutoff:  5741  3.100644
qtot up to nchloc:    8.00000259
atom: Maximum radius for chlocal:    3.10064
atom: Maximum radius for r*vlocal+2*Zval:    3.10064

KBgen: Kleinman-Bylander projectors: 
GHOST: No ghost state for L =  0
   l= 0   rc=  1.991366   el= -0.298497   Ekb=  3.842607   kbcos=  0.249762
GHOST: No ghost state for L =  1
   l= 1   rc=  2.019582   el= -0.060693   Ekb=  3.344331   kbcos=  0.150158
GHOST: No ghost state for L =  2
   l= 2   rc=  1.989365   el= -0.319649   Ekb=-12.634118   kbcos= -0.666320
GHOST: No ghost state for L =  3
   l= 3   rc=  2.015527   el=  0.019204   Ekb= -1.703638   kbcos= -0.006130

KBgen: Total number of Kleinman-Bylander projectors:  16
atom: -------------------------------------------------------------------------

atom: SANKEY-TYPE ORBITALS:

SPLIT: Orbitals with angular momentum L= 0

SPLIT: Basis orbitals for state 4s

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.010000 Ry
Split based on tail norm

   izeta = 1
                 lambda =    1.000000
                     rc =    7.489451
                 energy =   -0.286809
                kinetic =    0.275752
    potential(screened) =   -0.562561
       potential(ionic) =   -5.711143

SPLIT: Orbitals with angular momentum L= 2

SPLIT: Basis orbitals for state 3d

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.010000 Ry
Split based on tail norm

   izeta = 1
                 lambda =    1.000000
                     rc =    4.484389
                 energy =   -0.308779
                kinetic =    8.971931
    potential(screened) =   -9.280710
       potential(ionic) =  -18.259713

POLgen: Perturbative polarization orbital with L=  1

POLgen: Polarization orbital for state 4s
Split based on tail norm

   izeta = 1
                     rc =    7.489451
                 energy =    0.017858
                kinetic =    0.540235
    potential(screened) =   -0.522377
       potential(ionic) =   -5.278927
atom: Total number of Sankey-type orbitals:  9

atm_pop: Valence configuration (for local Pseudopot. screening):
 4s( 2.00)                                                            
 4p( 0.00)                                                            
 3d( 6.00)                                                            
Vna: chval, zval:    8.00000   8.00000

Vna:  Cut-off radius for the neutral-atom potential:   7.489451
comcore: Pseudo-core radius Rcore=  3.839063

atom: _________________________________________________________________________

prinput: Basis input 

PAO.BasisType split     

%block ChemicalSpeciesLabel
    1   26 Fe_nc                   # Species index, atomic number, species label
%endblock ChemicalSpeciesLabel

%block PAO.Basis                 # Define Basis set
# WARNING: This information might be incomplete!
Fe_nc                 2                    # Species label, number of l-shells
 n=4   0   1 P   1                   # n, l, Nzeta, Polarization, NzetaPol
   7.489   
   1.000   
 n=3   2   1                         # n, l, Nzeta 
   4.484   
   1.000   
%endblock PAO.Basis

prinput: ----------------------------------------------------------------------

Dumping basis to NetCDF file Fe_nc.ion.nc
coor:   Atomic-coordinates input format  =     Cartesian coordinates
coor:                                            (in Bohr units)

siesta: Atomic coordinates (Bohr) and species
siesta:      0.00000   0.00000   3.64400  1        1
siesta:      0.00000   0.00000   0.00000  1        2
siesta:      0.00000   0.00000  -3.64400  1        3

siesta: System type = bulk      

initatomlists: Number of atoms, orbitals, and projectors:      3    27    48

siesta: ******************** Simulation parameters ****************************
siesta:
siesta: The following are some of the parameters of the simulation.
siesta: A complete list of the parameters used, including default values,
siesta: can be found in file out.fdf
siesta:
redata: Spin configuration                          = non-collinear
redata: Number of spin components                   = 4
redata: Time-Reversal Symmetry                      = F
redata: Spin spiral                                 = F
redata: Long output                                 =   F
redata: Number of Atomic Species                    =        1
redata: Charge density info will appear in .RHO file
redata: Write Mulliken charges (when)               = geometry+end
redata: Write Mulliken Pop.                         = Atomic and Orbital charges
redata: Write Hirshfeld charges (when)              = none
redata: Write Voronoi charges (when)                = none
redata: Write spin charge (when)                    = scf
redata: Matel table size (NRTAB)                    =     1024
redata: Mesh Cutoff                                 =   100.0000 Ry
redata: Net charge of the system                    =     0.0000 |e|
redata: Min. number of SCF Iter                     =        0
redata: Max. number of SCF Iter                     =       20
redata: SCF mix quantity                            = Hamiltonian
redata: Mix DM or H after convergence               =   F
redata: Recompute H after scf cycle                 =   F
redata: Mix DM in first SCF step                    =   T
redata: Write Pulay info on disk                    =   F
redata: New DM Occupancy tolerance                  = 0.000000000001
redata: No kicks to SCF
redata: DM Mixing Weight for Kicks                  =     0.5000
redata: Require Harris convergence for SCF          =   F
redata: Harris energy tolerance for SCF             =     0.000100 eV
redata: Require DM convergence for SCF              =   T
redata: DM tolerance for SCF                        =     0.000100
redata: Require EDM convergence for SCF             =   F
redata: EDM tolerance for SCF                       =     0.001000 eV
redata: Require H convergence for SCF               =   T
redata: Hamiltonian tolerance for SCF               =     0.001000 eV
redata: Require (free) Energy convergence for SCF   =   F
redata: (free) Energy tolerance for SCF             =     0.000100 eV
redata: Using DFT-D3 dispersion                     =   F
redata: Using Saved Data (generic)                  =   F
redata: Use continuation files for DM               =   T
redata: Neglect nonoverlap interactions             =   F
redata: Method of Calculation                       = Diagonalization
redata: Electronic Temperature                      =   300.0000 K
redata: Fix the spin of the system                  =   F
redata: Max. number of TDED Iter                    =        1
redata: Number of TDED substeps                     =        3
redata: Dynamics option                             = Single-point calculation
mix.SCF: Pulay mixing                            = Pulay
mix.SCF:    Variant                              = stable
mix.SCF:    History steps                        = 8
mix.SCF:    Linear mixing weight                 =     0.010000
mix.SCF:    Mixing weight                        =     0.010000
mix.SCF:    SVD condition                        = 0.1000E-07
mix.SCF: Spin-component mixing                   all
redata: Save all siesta data in one NC              =   F
redata: ***********************************************************************

%block SCF.Mixers
  Pulay
%endblock SCF.Mixers

%block SCF.Mixer.Pulay
  # Mixing method
  method pulay
  variant stable

  # Mixing options
  weight 0.0100
  weight.linear 0.0100
  history 8
%endblock SCF.Mixer.Pulay

DM_history_depth set to one: no extrapolation allowed by default for geometry relaxation
Size of DM history Fstack: 1
Total number of electrons:    24.000000
Total ionic charge:    24.000000

siesta: k-grid: Number of k-points = 1
siesta: k-point is Gamma-only

diag: Algorithm                                     = D&C
diag: Used triangular part                          = Lower
diag: Absolute tolerance                            =  0.100E-15
diag: Orthogonalization factor                      =  0.100E-05
diag: Memory factor                                 =  1.0000


ts: **************************************************************
ts: Save H and S matrices                           =    F
ts: Save DM and EDM matrices                        =    F
ts: Only save the overlap matrix S                  =    F
ts: **************************************************************

************************ Begin: TS CHECKS AND WARNINGS ************************
************************ End: TS CHECKS AND WARNINGS **************************


                     ====================================
                        Single-point calculation
                     ====================================

outcell: Unit cell vectors (Ang):
        6.350127    0.000000    0.000000
        0.000000    6.350127    0.000000
        0.000000    0.000000   10.583544

outcell: Cell vector modules (Ang)   :    6.350127    6.350127   10.583544
outcell: Cell angles (23,13,12) (deg):     90.0000     90.0000     90.0000
outcell: Cell volume (Ang**3)        :    426.7720
Gamma-point calculation with interaction between periodic images
Some features might not work optimally:
e.g. DM initialization from atomic data
<dSpData1D:S at geom step 0
  <sparsity:sparsity for geom step 0
    nrows_g=27 nrows=27 sparsity=1.0000 nnzs=729, refcount: 7>
  <dData1D:(new from dSpData1D) n=729, refcount: 1>
refcount: 1>
new_DM -- step:     1
Initializing Density Matrix...

Attempting to read DM from file... Failed...
DM filled with atomic data (user-defined):
<dSpData2D:DM initialized from atoms
  <sparsity:sparsity for geom step 0
    nrows_g=27 nrows=27 sparsity=1.0000 nnzs=729, refcount: 8>
  <dData2D:DM n=729 m=4, refcount: 1>
refcount: 1>
No. of atoms with KB's overlaping orbs in proc 0. Max # of overlaps:       3      27

InitMesh: MESH = 40 x 40 x 64 = 102400
InitMesh: Mesh cutoff (required, used) =   100.000   101.065 Ry
New grid distribution [1]: sub = 2

stepf: Fermi-Dirac step function

siesta: Program's energy decomposition (eV):
siesta: Ebs     =       268.300351
siesta: Eions   =      2102.576618
siesta: Ena     =        86.583981
siesta: Ekin    =      2854.204183
siesta: Enl     =     -1849.754538
siesta: Eso     =         0.000000
siesta: Edftu   =         0.000000
siesta: DEna    =      -112.524750
siesta: DUscf   =        61.709365
siesta: DUext   =         0.000000
siesta: Ex      =     -1174.119038
siesta: Ec      =       -69.063535
siesta: Exc     =     -1243.182574
siesta: EbV     =         0.000000
siesta: eta*DQ  =         0.000000
siesta: Emadel  =         0.000000
siesta: Emeta   =         0.000000
siesta: Emolmec =         0.000000
siesta: Ekinion =         0.000000
siesta: Eharris =     -2221.670746
siesta: Etot    =     -2305.540951
siesta: FreeEng =     -2305.610193

        iscf     Eharris(eV)        E_KS(eV)     FreeEng(eV)     dDmax    Ef(eV) dHmax(eV)
   scf:    1    -2221.670746    -2305.540951    -2305.610193  0.988282 -8.716155 38.342280
     spin moment: {S} , |S| = {    1.87021    0.00000    0.00000 }     1.87021
timer: Routine,Calls,Time,% = IterSCF        1       0.094  20.90
   scf:    2    -2315.570287    -2310.807431    -2310.892745  0.118080 -8.578539 35.061346
     spin moment: {S} , |S| = {    1.77876   -0.00000   -0.00000 }     1.77876
   scf:    3    -2267.280842    -2317.942869    -2317.991410  0.605051 -7.157292 22.187660
     spin moment: {S} , |S| = {    3.54874    0.00000    0.00000 }     3.54874
   scf:    4    -2105.069025    -2274.381998    -2274.429396  0.904684 -9.110910 52.459027
     spin moment: {S} , |S| = {    0.52290   -0.00000   -0.00000 }     0.52290
   scf:    5    -2178.042714    -2233.933373    -2233.980691  0.427162 -5.914270 64.445847
     spin moment: {S} , |S| = {    0.08824   -0.00000    0.00000 }     0.08824
   scf:    6    -2146.453627    -2196.537265    -2196.537571  0.461392 -2.080058 54.853199
     spin moment: {S} , |S| = {    0.42644   -0.00000   -0.00000 }     0.42644
   scf:    7     -807.897254    -1975.330323    -1975.330323  0.996764 -1.892602 91.520636
     spin moment: {S} , |S| = {    0.01093    0.00000   -0.00000 }     0.01093
   scf:    8    -1923.142451    -2293.975278    -2293.975278  0.988189 -3.419643 33.842383
     spin moment: {S} , |S| = {    0.67861   -0.00000   -0.00000 }     0.67861
   scf:    9    -1852.764760    -2232.695540    -2232.695584  0.818121 -3.282186 50.466287
     spin moment: {S} , |S| = {    0.77481   -0.00000   -0.00000 }     0.77481
   scf:   10    -2024.077426    -2135.495986    -2135.495986  0.155917 -3.083690 67.871412
     spin moment: {S} , |S| = {    0.21865    0.00000   -0.00000 }     0.21865
   scf:   11    -2403.507044    -2295.055271    -2295.057502  0.350407 -3.570267 34.478560
     spin moment: {S} , |S| = {    1.53973   -0.00000   -0.00000 }     1.53973
   scf:   12    -2367.036228    -2342.450314    -2342.453416  0.232890 -3.691972 12.075033
     spin moment: {S} , |S| = {    1.97520   -0.00000   -0.00000 }     1.97520
   scf:   13    -2212.848901    -2312.330627    -2312.330639  0.380632 -3.965639 28.305779
     spin moment: {S} , |S| = {    0.77802    0.00000   -0.00000 }     0.77802
   scf:   14    -2142.715146    -2304.736791    -2304.748084  0.589138 -4.250872 30.061452
     spin moment: {S} , |S| = {    1.74263    0.00000   -0.00000 }     1.74263
   scf:   15    -2363.632287    -2343.341589    -2343.344573  0.219169 -4.048119 10.663869
     spin moment: {S} , |S| = {    1.97244    0.00000   -0.00000 }     1.97244
   scf:   16    -2212.373317    -2311.539384    -2311.539385  0.374493 -3.823711 28.294924
     spin moment: {S} , |S| = {    0.82938    0.00000   -0.00000 }     0.82938
   scf:   17    -2351.366434    -2333.976169    -2333.976169  0.107470 -3.680627 16.882499
     spin moment: {S} , |S| = {    1.27283    0.00000    0.00000 }     1.27283
   scf:   18    -2334.262203    -2334.119250    -2334.119250  0.008576 -3.665082 16.738960
     spin moment: {S} , |S| = {    1.21021    0.00000    0.00000 }     1.21021
   scf:   19    -2332.430126    -2333.280909    -2333.280909  0.026556 -3.284835 16.470234
     spin moment: {S} , |S| = {    0.97134    0.00000    0.00000 }     0.97134
   scf:   20    -2353.743132    -2347.344271    -2347.346406  0.141544 -4.268560  5.173634
     spin moment: {S} , |S| = {    1.20442    0.00000   -0.00000 }     1.20442
SCF_NOT_CONV: SCF did not converge  in maximum number of steps.
Geom step, scf iteration, dDmax, dHmax:    0    20      0.141544     0.380255

Using DM_out to compute the final energy and forces
No. of atoms with KB's overlaping orbs in proc 0. Max # of overlaps:       3      27

siesta: E_KS(eV) =            -2347.3443

siesta: E_KS - E_eggbox =     -2347.3443

siesta: Atomic forces (eV/Ang):
----------------------------------------
   Tot   -0.000000    0.000000    0.000000
----------------------------------------
   Max    3.623402
   Res    1.708088    sqrt( Sum f_i^2 / 3N )
----------------------------------------
   Max    3.623402    constrained

Stress tensor Voigt[x,y,z,yz,xz,xy] (kbar):        2.98        2.98      -34.71       -0.00       -0.00        0.00
(Free)E + p*V (eV/cell)    -2344.7948
Target enthalpy (eV/cell)    -2347.3464

mulliken: Atomic and Orbital Populations:

Species: Fe_nc               

Atom      Orb        Charge      Spin       Svec
----------------------------------------------------------------
    1  1 4s         1.34225   0.11889     -0.011   0.000   0.118
    1  2 3dxy       1.20171   0.79535      0.034   0.000   0.795
    1  3 3dyz       1.28823   0.57321     -0.004   0.000   0.573
    1  4 3dz2       1.36255   0.52884      0.006   0.000   0.529
    1  5 3dxz       1.28823   0.57321     -0.004   0.000   0.573
    1  6 3dx2-y2    1.20272   0.79438      0.034   0.000   0.794
    1  7 4Ppy       0.02114   0.00183      0.000   0.000  -0.002
    1  8 4Ppz       0.02658   0.01431      0.003  -0.000   0.014
    1  9 4Ppx       0.02114   0.00183      0.000   0.000  -0.002
    1     Total     7.75456   3.39269      0.059   0.000   3.392

    2  1 4s         1.01227   0.03719      0.037  -0.000   0.000
    2  2 3dxy       1.60139   0.31489      0.315   0.000   0.000
    2  3 3dyz       1.29658   0.12675      0.127  -0.000   0.000
    2  4 3dz2       1.28440   0.13386      0.134   0.000   0.000
    2  5 3dxz       1.29658   0.12675      0.127  -0.000   0.000
    2  6 3dx2-y2    1.59590   0.32307      0.323   0.000   0.000
    2  7 4Ppy       0.08467   0.00978      0.010  -0.000  -0.000
    2  8 4Ppz       0.23444   0.00468      0.005   0.000  -0.000
    2  9 4Ppx       0.08467   0.00978      0.010  -0.000  -0.000
    2     Total     8.49089   1.08673      1.087   0.000   0.000

    3  1 4s         1.34225   0.11889     -0.011   0.000  -0.118
    3  2 3dxy       1.20171   0.79535      0.034   0.000  -0.795
    3  3 3dyz       1.28823   0.57321     -0.004   0.000  -0.573
    3  4 3dz2       1.36255   0.52884      0.006   0.000  -0.529
    3  5 3dxz       1.28823   0.57321     -0.004   0.000  -0.573
    3  6 3dx2-y2    1.20272   0.79438      0.034   0.000  -0.794
    3  7 4Ppy       0.02114   0.00183      0.000   0.000   0.002
    3  8 4Ppz       0.02658   0.01431      0.003  -0.000  -0.014
    3  9 4Ppx       0.02114   0.00183      0.000   0.000   0.002
    3     Total     7.75456   3.39269      0.059   0.000  -3.392

----------------------------------------------------------------
Total              24.00000   1.20442      1.204   0.000  -0.000


Mulliken Atomic Populations:
Atom #   charge [q] valence [e]           S      Sx [e]      Sy [e]      Sz [e]  Species
     1     0.245443    7.754557    3.392689    0.058846    0.000000    3.392178  Fe_nc
     2    -0.490886    8.490886    1.086732    1.086732    0.000000    0.000000  Fe_nc
     3     0.245443    7.754557    3.392689    0.058846    0.000000   -3.392178  Fe_nc
-------------------------------------------------------------------------------
 Total     0.000000                1.204424    1.204424    0.000000   -0.000000

siesta: Program's energy decomposition (eV):
siesta: Ebs     =      -139.941037
siesta: Eions   =      2102.576618
siesta: Ena     =        86.583981
siesta: Ekin    =      2355.946858
siesta: Enl     =     -1496.236917
siesta: Eso     =         0.000000
siesta: Edftu   =         0.000000
siesta: DEna    =       -28.699963
siesta: DUscf   =         5.033853
siesta: DUext   =         0.000000
siesta: Ex      =     -1101.615848
siesta: Ec      =       -65.779617
siesta: Exc     =     -1167.395465
siesta: EbV     =         0.000000
siesta: eta*DQ  =         0.000000
siesta: Emadel  =         0.000000
siesta: Emeta   =         0.000000
siesta: Emolmec =         0.000000
siesta: Ekinion =         0.000000
siesta: Eharris =     -2353.743132
siesta: Etot    =     -2347.344271
siesta: FreeEng =     -2347.346406

siesta: Final energy (eV):
siesta:  Band Struct. =    -139.941037
siesta:       Kinetic =    2355.946858
siesta:       Hartree =    1410.632474
siesta:         Edftu =       0.000000
siesta:       Eso     =       0.000000
siesta:    Ext. field =       0.000000
siesta:         Exch. =   -1101.615848
siesta:         Corr. =     -65.779617
siesta:     Bulk bias =       0.000000
siesta:   Exch.-corr. =   -1167.395465
siesta:  Ion-electron =   -4754.925943
siesta:       Ion-ion =    -191.602195
siesta:       Ekinion =       0.000000
siesta: D3 dispersion =       0.000000
siesta:         Total =   -2347.344271
siesta:         Fermi =      -4.268560

siesta: Atomic forces (eV/Ang):
siesta:      1    0.000000    0.000000    3.623402
siesta:      2    0.000000    0.000000   -0.000000
siesta:      3   -0.000000   -0.000000   -3.623402
siesta: ----------------------------------------
siesta:    Tot   -0.000000    0.000000    0.000000

siesta: Stress tensor (static) (eV/Ang**3):
siesta:     0.001863   -0.000000   -0.000000
siesta:     0.000000    0.001863   -0.000000
siesta:    -0.000000   -0.000000   -0.021662

siesta: Cell volume =        426.771969 Ang**3

siesta: Pressure (static):
siesta:                Solid            Molecule  Units
siesta:           0.00006512         -0.00005376  Ry/Bohr**3
siesta:           0.00597896         -0.00493566  eV/Ang**3
siesta:           9.57935586         -7.90780439  kBar
(Free)E+ p_basis*V_orbitals  =       -2345.183733
(Free)Eharris+ p_basis*V_orbitals  =       -2351.582595

mulliken: Atomic and Orbital Populations:

Species: Fe_nc               

Atom      Orb        Charge      Spin       Svec
----------------------------------------------------------------
    1  1 4s         1.34225   0.11889     -0.011   0.000   0.118
    1  2 3dxy       1.20171   0.79535      0.034   0.000   0.795
    1  3 3dyz       1.28823   0.57321     -0.004   0.000   0.573
    1  4 3dz2       1.36255   0.52884      0.006   0.000   0.529
    1  5 3dxz       1.28823   0.57321     -0.004   0.000   0.573
    1  6 3dx2-y2    1.20272   0.79438      0.034   0.000   0.794
    1  7 4Ppy       0.02114   0.00183      0.000   0.000  -0.002
    1  8 4Ppz       0.02658   0.01431      0.003  -0.000   0.014
    1  9 4Ppx       0.02114   0.00183      0.000   0.000  -0.002
    1     Total     7.75456   3.39269      0.059   0.000   3.392

    2  1 4s         1.01227   0.03719      0.037  -0.000   0.000
    2  2 3dxy       1.60139   0.31489      0.315   0.000   0.000
    2  3 3dyz       1.29658   0.12675      0.127  -0.000   0.000
    2  4 3dz2       1.28440   0.13386      0.134   0.000   0.000
    2  5 3dxz       1.29658   0.12675      0.127  -0.000   0.000
    2  6 3dx2-y2    1.59590   0.32307      0.323   0.000   0.000
    2  7 4Ppy       0.08467   0.00978      0.010  -0.000  -0.000
    2  8 4Ppz       0.23444   0.00468      0.005   0.000  -0.000
    2  9 4Ppx       0.08467   0.00978      0.010  -0.000  -0.000
    2     Total     8.49089   1.08673      1.087   0.000   0.000

    3  1 4s         1.34225   0.11889     -0.011   0.000  -0.118
    3  2 3dxy       1.20171   0.79535      0.034   0.000  -0.795
    3  3 3dyz       1.28823   0.57321     -0.004   0.000  -0.573
    3  4 3dz2       1.36255   0.52884      0.006   0.000  -0.529
    3  5 3dxz       1.28823   0.57321     -0.004   0.000  -0.573
    3  6 3dx2-y2    1.20272   0.79438      0.034   0.000  -0.794
    3  7 4Ppy       0.02114   0.00183      0.000   0.000   0.002
    3  8 4Ppz       0.02658   0.01431      0.003  -0.000  -0.014
    3  9 4Ppx       0.02114   0.00183      0.000   0.000   0.002
    3     Total     7.75456   3.39269      0.059   0.000  -3.392

----------------------------------------------------------------
Total              24.00000   1.20442      1.204   0.000  -0.000


Mulliken Atomic Populations:
Atom #   charge [q] valence [e]           S      Sx [e]      Sy [e]      Sz [e]  Species
     1     0.245443    7.754557    3.392689    0.058846    0.000000    3.392178  Fe_nc
     2    -0.490886    8.490886    1.086732    1.086732    0.000000    0.000000  Fe_nc
     3     0.245443    7.754557    3.392689    0.058846    0.000000   -3.392178  Fe_nc
-------------------------------------------------------------------------------
 Total     0.000000                1.204424    1.204424    0.000000   -0.000000

cite: Please see "fe_noncol_gga.bib" for an exhaustive BiBTeX file.
cite: Please clearly indicate Siesta version in published work: 5.2.0-15-g8c00451d2
cite: This calculation has made use of the following articles
cite: which are encouraged to be cited in a published work.
        Primary SIESTA paper
          DOI: www.doi.org/10.1088/0953-8984/14/11/302


timer: Elapsed wall time (sec) =       2.420
timer: CPU execution times (sec):

Routine            Calls   Time/call    Tot.time        %
siesta                 1       2.419       2.419   100.00
Setup                  1       0.043       0.043     1.78
bands                  1       0.000       0.000     0.00
KSV_init               1       0.000       0.000     0.00
IterGeom               1       2.375       2.375    98.20
geom_init              1       0.314       0.314    12.99
state_init             1       0.024       0.024     0.98
hsparse                1       0.000       0.000     0.01
overlap                1       0.023       0.023     0.96
Setup_H0               1       0.201       0.201     8.30
naefs                  2       0.000       0.000     0.00
dnaefs                 2       0.000       0.000     0.00
MolMec                 2       0.000       0.000     0.00
kinefsm                2       0.011       0.023     0.94
nlefsm                 2       0.065       0.131     5.41
DHSCF_Init             1       0.048       0.048     1.98
DHSCF1                 1       0.006       0.006     0.24
InitMesh               1       0.006       0.006     0.23
INITMESH               1       0.000       0.000     0.00
DHSCF2                 1       0.042       0.042     1.74
REMESH                 1       0.003       0.003     0.12
REORD                196       0.000       0.017     0.71
PHION                  1       0.030       0.030     1.23
COMM_BSC             358       0.000       0.026     1.09
POISON                23       0.002       0.049     2.04
fft                   46       0.001       0.037     1.52
setup_H               21       0.093       1.949    80.56
DHSCF                 22       0.097       2.125    87.86
DHSCF3                22       0.093       2.040    84.33
rhoofd                22       0.011       0.244    10.09
XC                    22       0.067       1.473    60.88
GXC-CellXC            22       0.065       1.434    59.30
gridxc@cellXC         22       0.065       1.434    59.29
vmat                  22       0.009       0.204     8.42
IterSCF               20       0.094       1.880    77.73
compute_dm            20       0.001       0.017     0.71
diagon                20       0.001       0.017     0.70
cdiag                 20       0.001       0.014     0.57
cdiag1                20       0.000       0.001     0.06
cdiag2                20       0.000       0.002     0.07
cdiag3                20       0.000       0.009     0.36
cdiag4                20       0.000       0.001     0.06
MIXER                 20       0.000       0.002     0.07
PostSCF                1       0.180       0.180     7.45
FINAL_HF               1       0.180       0.180     7.45
DHSCF4                 1       0.085       0.085     3.52
dfscf                  1       0.076       0.076     3.12
overfsm                1       0.000       0.000     0.01
state_analysis         1       0.000       0.000     0.01
siesta_move            1       0.000       0.000     0.00
Analysis               1       0.000       0.000     0.01
optical                1       0.000       0.000     0.00
  

>> End of run:  17-JAN-2025  12:35:03
Job completed
